python/systems-manager/auto_patching/requirements.txt

```diff
@@ -0,0 +1 @@
+-e .
```

Comment:
Can we lift the contents of `python/systems-manager/auto_patching/*` one directory to `python/systems-manager`, and get rid of the `/auto_patching` folder?